### PR TITLE
Glass shards are bigger, and far less, crafting makes more sense

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
SUMMARY: Balance "Glass shards now have the old 250 ml volume. Crafting with it makes more sense. Also gets dropped far less"

#### Purpose of change
The new glass shards were broken. You could make a shiv out of 30 ml of a shard, which doesn't make a lot of sense at all. So in regard of old times sake, we revert to 250 ml per shard. And they drop far far less. So not as much clustering.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
<details>
  <summary>Full list of data files mentioning "glass_shard"</summary>
  
`$ grep glass_shard -r data`

```
data/mods/Salvaged_Robots/uncraft_corpse.json:      [ [ "glass_shard", 10 ] ]
data/mods/Modular_Turrets/monster_drops.json:      { "item": "glass_shard", "count": [ 0, 2 ] },
data/mods/DinoMod/mapgen/DinoLabFinale.json:            { "item": "glass_shard", "repeat": [ 25, 50 ], "chance": 100 },
data/mods/DinoMod/mapgen/DinoLabFinale.json:            { "item": "glass_shard", "repeat": [ 25, 50 ], "chance": 100 },
data/mods/DinoMod/mapgen/zoo.json:        { "item": "glass_shard", "repeat": [ 42, 84 ], "x": 10, "y": 22 },
data/mods/DinoMod/mapgen/zoo.json:        { "item": "glass_shard", "repeat": [ 42, 84 ], "x": 11, "y": [ 21, 23 ] }
data/mods/Aftershock/itemgroups/item_groups.json:      { "item": "glass_shard", "count": [ 0, 2 ] },
data/mods/Magiclysm/furniture.json:      "items": [ { "item": "scrap_copper", "count": [ 10, 20 ] }, { "item": "glass_shard", "count": [ 25, 50 ] } ]
data/mods/Magiclysm/furniture.json:        { "item": "glass_shard", "count": [ 8, 12 ] }
data/mods/No_Hope/Mapgen/necropolis.json:      { "item": "glass_shard", "prob": 15, "count": [ 1, 8 ] },
data/mods/No_Hope/Mapgen/necropolis.json:      { "item": "glass_shard", "prob": 10, "count": [ 1, 8 ] },
data/mods/No_Hope/Mapgen/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 14, "y": 16 },
data/mods/No_Hope/Mapgen/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 13, "y": 17 },
data/mods/No_Hope/Mapgen/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 14, "y": 17 },
data/mods/No_Hope/Mapgen/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 13, "y": 18 }
data/mods/No_Hope/Mapgen/2storymodern01.json:        { "item": "glass_shard", "x": 11, "y": 18, "chance": 50, "repeat": [ 5, 8 ] },
data/json/itemgroups/oa_shared_item_groups.json:      { "item": "glass_shard", "prob": 30, "count": [ 0, 1 ] },
data/json/itemgroups/Locations_MapExtras/mansion.json:      { "item": "glass_shard", "prob": 50, "count": [ 0, 5 ] },
data/json/itemgroups/Locations_MapExtras/mansion.json:      { "item": "glass_shard", "prob": 30, "count": [ 0, 1 ] },
data/json/itemgroups/Locations_MapExtras/mansion.json:      { "item": "glass_shard", "prob": 30, "count": [ 0, 1 ] },
data/json/itemgroups/Locations_MapExtras/mansion.json:      { "item": "glass_shard", "prob": 30, "count": [ 0, 1 ] },
data/json/itemgroups/Locations_MapExtras/locations.json:      { "item": "glass_shard", "prob": 100, "count-min": 2, "count-max": 10 },
data/json/itemgroups/Locations_MapExtras/locations.json:      { "item": "glass_shard", "prob": 650, "count": [ 0, 1 ] },
data/json/itemgroups/Locations_MapExtras/mall_item_groups.json:      { "item": "glass_shard", "prob": 10, "count": [ 0, 1 ] },
data/json/itemgroups/roof.json:      { "item": "glass_shard", "prob": 20, "count": [ 0, 1 ] },
data/json/itemgroups/trash_and_debris.json:      { "item": "glass_shard", "prob": 3, "count-min": 0, "count-max": 8 },
data/json/itemgroups/trash_and_debris.json:      { "item": "glass_shard", "prob": 20, "count": [ 0, 1 ] },
data/json/itemgroups/trash_and_debris.json:      [ "glass_shard", 10 ],
data/json/itemgroups/shops_trades.json:      { "item": "glass_shard", "prob": 90, "count": [ 0, 1 ] },
data/json/itemgroups/food_service.json:      { "item": "glass_shard", "prob": 30, "count": [ 0, 1 ] },
data/json/obsoletion/migrate_to_count_by_charges.json:    "list": [ "glass_shard", "FMCNote", "RobofacCoin", "FlatCoin", "signed_chit", "icon" ]
data/json/vehicleparts/windshields.json:    "breaks_into": [ { "count": [ 25, 50 ], "item": "glass_shard" } ],
data/json/vehicleparts/windshields.json:    "breaks_into": [ { "count": [ 25, 50 ], "item": "glass_shard" }, { "count": [ 4, 8 ], "item": "wire" } ],
data/json/vehicleparts/mirrors.json:    "breaks_into": [ { "item": "glass_shard", "count": [ 0, 1 ] } ]
data/json/vehicleparts/mirrors.json:    "breaks_into": [ { "item": "glass_shard", "count": [ 0, 1 ] } ]
data/json/recipes/ammo/shot.json:        [ "glass_shard", 1 ]
data/json/recipes/ammo/other.json:    "components": [ [ [ "glass_shard", 1 ], [ "flask_glass", 1 ], [ "test_tube", 2 ] ] ]
data/json/recipes/weapon/cutting.json:    "components": [ [ [ "glass_shard", 1 ] ], [ [ "rag", 1 ], [ "duct_tape", 10 ] ] ]
data/json/recipes/other/containers.json:    "components": [ [ [ "glass_shard", 5 ], [ "flask_glass", 5 ], [ "test_tube", 10 ], [ "marble", 125 ] ] ]
data/json/recipes/other/containers.json:    "components": [ [ [ "glass_shard", 3 ], [ "pipe_glass", 1 ], [ "flask_glass", 3 ], [ "test_tube", 6 ], [ "marble", 75 ] ] ]
data/json/recipes/other/containers.json:    "components": [ [ [ "glass_shard", 2 ], [ "flask_glass", 2 ], [ "test_tube", 4 ], [ "marble", 50 ] ] ]
data/json/recipes/other/containers.json:    "components": [ [ [ "glass_shard", 1 ], [ "test_tube", 2 ], [ "marble", 25 ] ] ]
data/json/recipes/other/containers.json:    "components": [ [ [ "glass_shard", 1 ], [ "flask_glass", 1 ], [ "marble", 25 ] ] ]
data/json/recipes/other/tools.json:    "components": [ [ [ "glass_shard", 3 ], [ "bottle_glass", 1 ], [ "flask_glass", 3 ], [ "test_tube", 6 ], [ "marble", 75 ] ] ]
data/json/recipes/other/traps.json:    "components": [ [ [ "glass_shard", 4 ] ], [ [ "superglue", 1 ], [ "bone_glue", 1 ] ] ]
data/json/recipes/other/traps.json:    "result": "glass_shard_trap",
data/json/recipes/other/traps.json:    "components": [ [ [ "glass_shard", 1 ] ] ]
data/json/recipes/other/materials.json:    "result": "glass_shard",
data/json/recipes/other/materials.json:    "components": [ [ [ "glass_shard", 18 ], [ "jar_glass", 9 ], [ "flask_glass", 18 ], [ "test_tube", 36 ] ] ]
data/json/recipes/other/materials.json:    "components": [ [ [ "glass_shard", 40 ], [ "flask_glass", 12 ], [ "jar_glass", 4 ] ] ]
data/json/recipes/other/materials.json:    "components": [ [ [ "glass_shard", 2 ] ], [ [ "chem_sulphur", 1 ] ], [ [ "scrap", 1 ] ], [ [ "charcoal", 1 ] ] ]
data/json/recipes/recipe_deconstruction.json:    "components": [ [ [ "glass_shard", 4 ] ] ]
data/json/recipes/recipe_deconstruction.json:    "components": [ [ [ "glass_shard", 4 ] ] ]
data/json/recipes/recipe_deconstruction.json:    "components": [ [ [ "glass_shard", 4 ] ] ]
data/json/recipes/recipe_deconstruction.json:    "components": [ [ [ "glass_shard", 182 ] ] ]
data/json/traps.json:    "drops": [ "glass_shard" ],
data/json/furniture_and_terrain/furniture-tools.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/furniture-tools.json:      "items": [ { "item": "glass_shard", "count": [ 1, 3 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-doors.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/furniture-decorative.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/furniture-decorative.json:        { "item": "glass_shard", "count": [ 0, 1 ] },
data/json/furniture_and_terrain/terrain-floors_indoor.json:      "items": [ { "item": "glass_shard", "count": [ 0, 2 ] } ]
data/json/furniture_and_terrain/furniture-zztesting.json:        { "item": "glass_shard", "count": [ 0, 2 ] },
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/terrain-walls.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "wire", "prob": 20 }, { "item": "scrap", "count": [ 3, 5 ] } ]
data/json/furniture_and_terrain/furniture-storage.json:        { "item": "glass_shard", "count": [ 1, 10 ] },
data/json/furniture_and_terrain/furniture-storage.json:      "items": [ { "item": "glass_shard", "count": [ 1, 5 ] } ]
data/json/furniture_and_terrain/furniture-recreation.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 5 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 5 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 3 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 3 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 3 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 3 ] },
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 5 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 0, 1 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "splinter", "count": [ 0, 2 ] }, { "item": "glass_shard", "count": [ 1, 4 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 1, 5 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 5 ] },
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 1, 5 ] },
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 5, 12 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 5, 12 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 5, 12 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 5, 12 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 5, 12 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 10, 20 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 10, 20 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 10, 20 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 10, 20 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 10, 20 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 10, 20 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 10, 20 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 10, 20 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:        { "item": "glass_shard", "count": [ 3, 8 ] }
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-windows.json:      "items": [ { "item": "glass_shard", "count": [ 3, 8 ] } ]
data/json/furniture_and_terrain/terrain-manufactured.json:      "items": [ { "item": "glass_shard", "count": [ 3, 6 ] }, { "item": "scrap", "count": [ 0, 2 ] } ]
data/json/furniture_and_terrain/terrain-fences-gates.json:        { "item": "glass_shard", "count": [ 5, 15 ] },
data/json/furniture_and_terrain/furniture-medical.json:        { "item": "glass_shard", "count": [ 3, 6 ] },
data/json/furniture_and_terrain/furniture-medical.json:        { "item": "glass_shard", "count": [ 2, 5 ] },
data/json/furniture_and_terrain/furniture-medical.json:        { "item": "glass_shard", "count": [ 2, 5 ] },
data/json/furniture_and_terrain/furniture-medical.json:        { "item": "glass_shard", "count": [ 2, 5 ] },
data/json/furniture_and_terrain/furniture-medical.json:        { "item": "glass_shard", "count": [ 2, 5 ] },
data/json/furniture_and_terrain/furniture-plumbing.json:        { "item": "glass_shard", "count": [ 1, 2 ] }
data/json/furniture_and_terrain/furniture-appliances.json:        { "item": "glass_shard", "count": [ 2, 4 ] },
data/json/items/resources/glass.json:    "id": "glass_shard",
data/json/items/resources/glass.json:    "ammo_type": "glass_shard",
data/json/items/ammo_types.json:    "id": "glass_shard",
data/json/items/ammo_types.json:    "default": "glass_shard"
data/json/items/tool/traps.json:    "id": "glass_shard_trap",
data/json/items/tool/traps.json:    "looks_like": "glass_shard",
data/json/mapgen/riverside/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 14, "y": 16 },
data/json/mapgen/riverside/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 13, "y": 17 },
data/json/mapgen/riverside/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 14, "y": 17 },
data/json/mapgen/riverside/whaleys_house_river.json:        { "item": "glass_shard", "repeat": 8, "x": 13, "y": 18 }
data/json/mapgen/necropolis/necropolis.json:      { "item": "glass_shard", "prob": 15, "count": [ 0, 1 ] },
data/json/mapgen/necropolis/necropolis.json:      { "item": "glass_shard", "prob": 10, "count": [ 0, 1 ] },
data/json/mapgen/lab/lab_escape.json:        "!": { "item": [ { "item": "glass_shard", "amount": [ 2, 5 ] } ] },
data/json/mapgen/house/2storymodern01.json:        { "item": "glass_shard", "x": 11, "y": 18, "chance": 100, "repeat": [ 5, 8 ] },
data/json/mapgen/zoo.json:        { "item": "glass_shard", "repeat": [ 42, 84 ], "x": 10, "y": 22 },
data/json/mapgen/zoo.json:        { "item": "glass_shard", "repeat": [ 42, 84 ], "x": 11, "y": [ 21, 23 ] }
data/json/mapgen/shelter.json:      [ "glass_shard", 80 ],
data/json/materials.json:    "burn_products": [ [ "glass_shard", 1 ] ]
data/json/construction.json:    "components": [ [ [ "glass_shard", 20 ] ] ],
data/json/construction.json:    "byproducts": [ { "item": "glass_shard", "count": [ 1, 3 ] } ],
```
  
</details>

There are still some unchanged ones.
Notable examples:
* 3L glass jar has wrong number
* Glass sheet craft then uncraft
* Mods - non-obsoleted mods should still be updated
* Mapgen with `repeat`
* 3 hardcoded entries: two in `map.cpp` regarding breaking furniture, one in `mapgen.cpp` for broken vending machines

**Comment by Lamandus** [comment]
Error in Auto blueprint doesn't make any sense. I didn't change any of these recipes and the recipes are the same as adviced.

File: data/json/construction.json [hunk 1]
```diff
@@ -2218,7 +2201,6 @@
     "type": "construction",
     "id": "constr_window_bars_curtains",
     "description": "Install Bars Onto Window",
-    "pre_note": "Install Curtains Onto Window With Bars",
```
Comment (Coolthulhu) [inline data/json/construction.json:2221]: Why remove?

File: data/json/construction.json [hunk 2]
```diff
@@ -2204,7 +2188,6 @@
     "type": "construction",
     "id": "constr_window_bars_alarm",
     "description": "Install Bars Onto Window",
-    "pre_note": "Install Bars Onto Store Showcase Window With Alarm",
```
Comment (Coolthulhu) [inline data/json/construction.json:2207]: And here

File: data/json/construction.json [hunk 3]
```diff
@@ -2172,25 +2171,10 @@
...
-    "using": [ [ "welding_standard", 4 ] ],
-    "components": [ [ [ "rebar", 8 ] ], [ [ "sheet_metal_small", 6 ] ] ],
-    "pre_terrain": "t_window_empty",
-    "post_terrain": "t_window_bars"
-  },
   {
     "type": "construction",
     "id": "constr_window_bars_no_curtains",
     "description": "Install Bars Onto Window",
-    "pre_note": "Install Bars Onto Window With No Curtains",
```
Comment (Coolthulhu) [inline data/json/construction.json:2193]: and here

File: data/json/construction.json [hunk 4]
```diff
@@ -2172,25 +2171,10 @@
...
-    "pre_note": "Install Bars Onto Empty Window Frame",
-    "category": "WINDOWS",
-    "required_skills": [ [ "fabrication", 5 ] ],
-    "time": "90 m",
-    "qualities": [ [ { "id": "SAW_M", "level": 1 } ], [ { "id": "SCREW", "level": 1 } ] ],
-    "using": [ [ "welding_standard", 4 ] ],
-    "components": [ [ [ "rebar", 8 ] ], [ [ "sheet_metal_small", 6 ] ] ],
-    "pre_terrain": "t_window_empty",
-    "post_terrain": "t_window_bars"
-  },
```
Comment (Coolthulhu) [inline data/json/construction.json:2188]: and most importantly here

File: data/json/construction.json [hunk 5]
```diff
@@ -2162,7 +2162,6 @@
     "type": "construction",
     "id": "constr_window_bars",
     "description": "Install Bars Onto Window",
-    "pre_note": "Install Bars Onto Store Showcase Window",
```
Comment (Coolthulhu) [inline data/json/construction.json:2165]: Oh, there's one here, but it's just a note.

File: data/json/construction.json [hunk 6]
```diff
@@ -2042,7 +2042,7 @@
     "id": "constr_covered_well",
     "description": "Build Water Well",
     "category": "CONSTRUCT",
-    "required_skills": [ [ "fabrication", 4 ] ],
+    "required_skills": [ [ "fabrication", 4 ], [ "survival", 4 ] ],
```
Comment (Coolthulhu) [inline data/json/construction.json:2045]: Seems like you have old version of the file and somehow committed the changes to make things back to old.

File: data/json/items/melee/bludgeons.json [hunk 7]
```diff
@@ -379,7 +379,7 @@
     "material": [ "glass", "cotton" ],
     "weight": "115 g",
     "price_postapoc": 10,
-    "volume": "30 ml",
+    "volume": "250 ml",
```
Comment (Coolthulhu) [inline data/json/items/melee/bludgeons.json:382]: Not related to this PR at all, but I just realized that glass shiv is in a file named `bludgeons.json`.

**Comment by Coolthulhu** [comment]
There are still some files without changes, like `data/json/mapgen/house/2storymodern01.json`.
In cases where the number can't be decreased, the chance or repeat for mapgen item placement should be lowered proportionally.
The way it works is that the spawn is rolled `repeat` times and every roll the chance to actually spawn is `chance*100%*world_loot_modifier`.

**Comment by Lamandus** [comment]
> There are still some files without changes, like `data/json/mapgen/house/2storymodern01.json`.
> In cases where the number can't be decreased, the chance or repeat for mapgen item placement should be lowered proportionally.
> The way it works is that the spawn is rolled `repeat` times and every roll the chance to actually spawn is `chance*100%*world_loot_modifier`.

please take another look, it should be good now for real.

**Comment by Coolthulhu** [comment]
Has a tiny conflict now
I can fix it on my side, but then I'll need to leave it for the end of the merge run.

**Comment by Lamandus** [comment]
> Has a tiny conflict now I can fix it on my side, but then I'll need to leave it for the end of the merge run.

No problem, Take your time, I would appreciate it. If you can do it.

Thank you

**Pull request merged**